### DISCUSSION TITLE
Have you considered supporting machine learning?

### DISCUSSION THREAD

**Discussion by jet10000** [supabase/supabase#7537] [Feature Requests]
perform machine learning in sql

**Comment by burggraf** [comment]
Do you have any specific tools in mind for this?

**Reply by jet10000** [reply]
Like this: https://postgresml.org/

**Reply by holmofy** [reply]
[Madlib](https://madlib.apache.org/) is also good

**Reply by data-envoy** [reply]
MADlib only has support up to python 2.7, check: 
https://cwiki.apache.org/confluence/display/MADLIB/Installation+Guide & https://www.mail-archive.com/issues@madlib.apache.org/msg02684.html
I doubt this would be a good fit for Supabase who seem to be on the bleeding edge of updates.
